tools/generate-helpers/pg-table-bindings/store.go.tpl

```diff
@@ -210,11 +226,24 @@ func (s *storeImpl) {{ template "copyFunctionName" $schema }}(ctx context.Contex
             {{ range $idx, $field := $schema.DBColumnFields }}
             {{if eq $field.DataType "datetime"}}
             pgutils.NilOrTime({{$field.Getter "obj"}}),
+            {{- else if eq $field.SQLType "uuid" }}
+            pgutils.NilOrUUID({{$field.Getter "obj"}}),
             {{- else}}
             {{$field.Getter "obj"}},{{end}}
             {{end}}
         })
 
+        {{- if $migration }}
+        {{- range $field := $schema.PrimaryKeys -}}
+            {{- if eq $field.SQLType "uuid" }}
+            if pgutils.NilOrUUID({{$field.Getter "obj"}}) == nil {
+                log.Infof("{{$field.Name}} is not a valid uuid -- %v", obj)
```

Comment:
Can you add a utils.Should here so that we can catch potential issue in dev environment.